portality/templates/application_form/js/_form_js.html

```diff
@@ -29,6 +29,9 @@
             {% if form_diff %}
                 formDiff: {{ form_diff|tojson }},
             {% endif %}
+            {% if obj.application_status == "update_request" and (not current_journal or not current_journal.is_in_doaj()) %}
+                withdrawn: "true"
+            {% endif %}
```

Comment:
Same comments as above about determining if `withdrawn` is true.  Also, why is `"true"` a string here?  Should be a boolean